lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -106,6 +113,45 @@ def has_instructor_tab(user, course):
         student = UserFactory.create()
         self.assertFalse(has_instructor_tab(student, self.course))
 
+    @ddt.data(
+        ("How to defeat the Road Runner", "2017", "001", "ACME"),
+    )
+    @ddt.unpack
+    def test_instructor_course_info(self, display_name, run, number, org):
+        """
+        Verify that it shows the correct course information
+        """
+        url =\
```

Comment:
We generally don't break our lines with a `\`.  Instead, put the call to `reverse` on the same line as `url =`.